pelican/writers.py

```diff
@@ -83,13 +82,17 @@ def _add_item_to_the_feed(self, feed, item):
             unique_id=get_tag_uri(link, item.date),
             description=description,
             content=content,
-            categories=categories if categories else None,
+            categories=categories or None,
```

Comment:
Personally, I think
```python
a if a else b
```
is easier to parse than
```python
a or b
```

Reply:
Somehow if we want to make the shortcode to keep it clean and understood.